### PR TITLE
Add comment text

### PR DESCRIPTION
**Pull request by sli1989**
## PR Checklist
**Please check if your PR fulfills the following requirements:**

- [x] The commit message follows [our guidelines](https://github.com/theme-next/hexo-theme-next/blob/master/.github/CONTRIBUTING.md).
- [x] Tests for the changes have been added (for bug fixes / features).
   - [x] Muse | Mist have been tested.
   - [x] Pisces | Gemini have been tested.
- [x] Docs have been added / updated (for bug fixes / features).

## PR Type
**What kind of change does this PR introduce?**  

- [ ] Bugfix.
- [x] Feature.
- [ ] Code style update (formatting, local variables).
- [ ] Refactoring (no functional changes, no api changes).
- [ ] Build related changes.
- [ ] CI related changes.
- [ ] Documentation content changes.
- [ ] Other... Please describe:

## What is the current behavior?

At present, there is a loss in comment text.

Facebook comments has added already ([example](https://www.lavicle.com/2017/06/10/hello-world/)). 

https://github.com/sli1989/hexo-theme-next/blob/e6d4dd5ad7f178708754e0bab01863f267c7a900/layout/_macro/post.swig#L139

Disqus auto return the `Comments`: [1 Comment example](https://wafer.li/Buying/Realforce%2087u/), [2 Comments ](https://wafer.li/Kotlin/Kotlin%20%E5%BC%82%E5%B8%B8/), [6 Comments ](https://wafer.li/Kotlin/Kotlin%20%E5%AF%B9%E8%B1%A1/)

Hypercomments ([example](https://almostover.ru/2017-04/advanced-ssl-tuning/)) & changyan ([example](http://www.zhaiqianfeng.com/2017/03/changyan-for-hexo-next-theme.html)) & valine & gitment are needed to add comment text.

gitment example

Issue Number(s): N/A

## What is the new behavior?

To keep consisting of disqus comment (auto display), the style of `commentCount+Comments` in `<a </a>` as hyperlink is added in hypercomments/changyan/gitment/valine. And change `comments` to `Comments` in facebook comments.

* Screens with this changes: 

valine

* Link to demo site with this changes: http://test.saili.science/2018/03/18/hello-world/

### How to use?
In NexT `_config.yml`:
```yml
...
```

## Does this PR introduce a breaking change?
- [ ] Yes.
- [x] No.

### CODE REVIEW

**Comment by ivan-nginx** [comment]
No. In some comments (e.g. Disqus) label «Comment(s)» added automatically.

**Comment by sli1989** [comment]
i have thought about disqus as exception. facebook and changyan comment are needed to check.
there is a abnormal indent...😂

**Comment by sli1989** [comment]
Confirm in all existed comment systems.

File: layout/_macro/post.swig [hunk 1]
```diff
@@ -136,7 +136,7 @@
                   <i class="fa fa-comment-o"></i>
                 </span>
                 <a href="{{ url_for(post.path) }}#comments" itemprop="discussionUrl">
-                  <span class="post-comments-count fb-comments-count" data-href="{{ post.permalink }}" itemprop="commentCount">0</span> comments
+                  <span class="post-comments-count fb-comments-count" data-href="{{ post.permalink }}" itemprop="commentCount">0</span> Comments
```
Comment (ivan-nginx) [inline layout/_macro/post.swig]: What about here? It is not translated text?

Comment (sli1989) [reply layout/_macro/post.swig]: oh, it's a mistake. I will fix it.

**Comment by sli1989** [comment]
> If mobile style - icon only, without label.
At present, it's follow the display rules of Post meta display settings. The text will not display in mobile.

https://github.com/theme-next/hexo-theme-next/blob/1975fd0c95e85abad045c7f1dd2dd67e0e2c09a1/_config.yml#L250-L257

https://github.com/theme-next/hexo-theme-next/blob/9d46733e2f9b776dfe1bab5e65be0937dd4ca5b8/source/css/_common/components/post/post-meta.styl#L38-L45

> And second: 1 comment, 2 comment(s) — how about this?

The disqus comment returns '* Comments' automatically. I think it's hard to unify them. Other comment systems can suit the singular and plural forms according to the tags/categories count.

https://github.com/theme-next/hexo-theme-next/blob/1975fd0c95e85abad045c7f1dd2dd67e0e2c09a1/languages/_en.yml#L56-L60

**Comment by ivan-nginx** [comment]
> The disqus comment returns '* Comments' automatically. I think it's hard to unify them. Other comment systems can suit the singular and plural forms according to the tags/categories count.

Yes, it is. I'm talking about this at the begining of this pull. And impossible to resolve how many comments count will and there is bug in languages:

1. 0 comments
    1 ~~comments~~
    5 comments
2. 0 комментариев
    1 ~~комментарий~~
    2 ~~комментария~~
    5 комментариев

About Chines IDK, but i think there is 2 variants, like in EN language.

So, how about this?

**Comment by sli1989** [comment]
I don't know how to get the element of comment count, for valine: `<span class="post-comments-count valine-comment-count" data-xid="{{ url_for(post.path) }}" itemprop="commentCount"></span>`...

**Comment by ivan-nginx** [comment]
Nohow. :smile: This is 3rd-party JS applications, NexT can't resolve this data during generate.

***

I suggest something like:
`Comments: X`

So, it will be: 

1. Comments: 0
    Comments: 1
    Comments: 5
2. Комментариев: 0
    Комментариев: 1
    Комментариев: 2
    Комментариев: 5

**Comment by sli1989** [comment]
The comment number can be counted like this?

https://github.com/theme-next/hexo-theme-next/blob/95cd9e1c856d5761d48bc0fcae8653ab170c2a43/layout/page.swig#L33-L41

**Comment by ivan-nginx** [comment]
They can be counted like this, yep. But only if it's internal NexT or Hexo plugin(s). Comments services are external.

**Comment by ivan-nginx** [comment]
MB symbol `:` from tranlsate?

**Comment by sli1989** [comment]
OK, set the text in or out of hyperlink ?

**Comment by ivan-nginx** [comment]
Replace `&#58` with `{{ __('symbol.colon') }}`.

**Comment by sli1989** [comment]
I say the style of text. Comments: [1](url) or [Comments: 1](url)

**Comment by ivan-nginx** [comment]
Oh. IDK. See as will be better. FB comments was out of the link.

**Comment by ivan-nginx** [comment]
Let's see how in Wordpress?

**Comment by sli1989** [comment]
Within the hyperlink. That's it.

**Comment by ivan-nginx** [comment]
Yep, [i see](http://test.saili.science/). Nice!
image

**Pull request merged**